### PR TITLE
Add stubs to peteneuwald Milwaukee file

### PR DESCRIPTION
**Pull request by peteneuwald**
Added stubs

### CODE REVIEW

File: peteneuwald-milwaukee.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Milwaukee, WI
+
+### Food Recommendations
+- City Tacos
```
Comment (ncunningcivica) [inline peteneuwald-milwaukee.md]: Milkwaukee must have the same Food places as San Diego 😄

Comment (peteneuwald) [reply peteneuwald-milwaukee.md]: Ha ha! Maybe.

**Review by peteneuwald — COMMENTED** [review]
Looks much better. 👍

**Review by ncunningcivica — APPROVED** [review]
Looks Great I feel I might actually get feed in Milwaukee if i'm ever there 👍

**Pull request merged**